src/blake.js

```diff
@@ -21,12 +22,21 @@ const blake2s = {
 // the function as async because it must return a Promise to match the API
 // for other functions that do perform asynchronous work (see sha.browser.js)
 // eslint-disable-next-line
+
+/**
+ * @param {number} size
+ * @param {any} hf
```

Comment:
nit: perhaps useful to type hf?

Reply:
yes but im going to defer that to another PR

Reply:
https://github.com/multiformats/js-multihashing-async/issues/96